pipelines/shopify/incremental_loading_helpers.py

```diff
@@ -0,0 +1,97 @@
+from datetime import datetime, timedelta, timezone
+from dateutil import parser
+from dlt.common.configuration import with_config
+from enum import Enum
+from time import sleep
+from typing import Optional
+
+import dlt
+import re
+
+
+class RelationType(str, Enum):
+    NEXT = 'next'
+    PREVIOUS = 'previous'
+
+
+def get_linked_url(link_header: str, relation_type: str) -> str:
```

Comment:
`requests` do that automatically. see `response.links` and https://github.com/dlt-hub/pipelines/blob/master/pipelines/github/__init__.py#L239